test/test_quantized.py

```diff
@@ -1071,6 +1071,32 @@ def equal_ref(qX, qX2):
         self.assertEqual(qX.equal(qX), equal_ref(qX, qX))
         self.assertEqual(qX.equal(qX2), equal_ref(qX, qX2))
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=4,
```

Comment:
Can we also test for qx being of type qint8?

Reply:
Added an assertion for type other than uint8. Updated test to use both qengines.

I'm not sure how we can make sure to test the AVX2 and default path as it is guarded by compile time macro. I am relying on CI to run on machines without AVX2.